caracal/sample_configurations/workflow_new.yml

```diff
@@ -0,0 +1,197 @@
+schema_version: 1.0.1
```

Comment:
Did you recommit this file again for some reason, or was it brought back because you edited it in your branch?